src/app/group-list/group-list.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit } from '@angular/core';
-import { LinkService } from '../link.service';
+import { ChromeAPIService } from '../chrome-api.service';
```

Comment:
I wonder if we can find something between "ChromeApiService" and "LinkService". Both are not quite capture the intention. It's not general enough to be called "ChromeApiService" since it's mainly deal with links. 

I'm thinking something link "ChromeLinkSubscriberService" but I'm not a big fan. I don't have a strong preference here, so up to you.